db/tgif.sql

```diff
@@ -68,4 +69,15 @@ VALUES
     (TRUE, 'Teresa'),
     (FALSE, 'Warren'),
     (FALSE, 'Ryan'),
-    (FALSE, 'Mentored');
\ No newline at end of file
+    (FALSE, 'Mentored');
+
+INSERT INTO proposals(title, organization, amount_requested, link, description_text)
+VALUES
+    ('treeees', 'tgif', 19.2, 'link____', 'desc');
+
+INSERT INTO comments(id, time_posted, comment_text, user_id, proposal_id)
```

Comment:
These comments aren't showing up in your database because some of the values you are inserting are not valid (i.e. user_id 33 doesn't exist so you cant insert a comment with user_id 33). Also, timestamp is a special datatype, not a string. If you want to have this dummy data, make sure to fix these values!